django_celery_tracker/templates/celery_tracker/timeline_dashboard.html

```diff
@@ -0,0 +1,74 @@
+{% extends "admin/base_site.html" %}
+{% load i18n static %}
+
+{% block title %}Django Celery Tracker{% endblock %}
+
+{% block branding %}
+  <h1>Django Celery Tracker</h1>
+{% endblock %}
+
+{% block extrahead %}
+  <link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/vis/4.21.0/vis.min.css">
+  <link rel="stylesheet" type="text/css" href="{% static 'admin/css/forms.css' %}">
+  <link rel="stylesheet" type="text/css" href="{% static 'celery_tracker/css/dashboard.css' %}">
+  <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+  <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/vis/4.21.0/vis.min.js"></script>
+  <script type="text/javascript" src="{% static 'celery_tracker/js/dashboard.js' %}"></script>
```

Comment:
The dashboard script uses `momentjs` but I don't see that imported here, is moment pulled in as a dependency by another script?

Reply:
Yup. The `vis.js` library exposes it via `vis.moment`.